homeassistant/components/sunricher_dali/config_flow.py

```diff
@@ -28,12 +29,100 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional("refresh", default=False): bool,
+    }
+)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
```

Comment:
This shouldn't be an Options flow.
1. we have a reconfigure flow to reconfigure the device. But, the device should be the same, so it should have the same serial number.
2. I don't see why people need to use this to rediscover devices? As in, when I reload the integration it would add the devices we find right? In theory it would be nice if we could add them on runtime, but we shouldn't have the user do this manually